front/pages/poke/[wId]/conversations/[cId]/index.tsx

```diff
@@ -237,35 +237,32 @@ interface ContentFragmentViewProps {
 
 const ContentFragmentView = ({ message }: ContentFragmentViewProps) => {
   return (
-    <div className="w-full">
-      <ConversationMessage type="system">
```

Comment:
Here change is from poké I removed the call to ConversationMessage typed as system that does not exist and replaces just text-sm and it looks similar.